artifacts/manifests/k8s-staging-sig-storage/master.yaml

```diff
@@ -0,0 +1,3 @@
+files:
+- name: 1.0.2/csi-proxy.exe
```

Comment:
Are there different architectures for windows binaries?

Reply:
I think we only support amd64 for Windows.

Reply:
Do we want to add the architecture to the file name in case we support more architectures in the future?

Reply:
there is no plan to support yet. But I think it is better to add it for future.